.circle/packagecloud.sh

```diff
@@ -178,6 +179,17 @@ function publish() {
   package_cloud push ${PACKAGECLOUD_ORGANIZATION}/${PACKAGECLOUD_REPO}/${PKG_OS_NAME}/${PKG_OS_VERSION} ${PKG_PATH} || exit 0
 }
 
+function prune_old_revision() {
+  if [ "$PKG_RELEASE" -gt "$MAX_REVISIONS" ]; then
+    REVISION_TO_DELETE=$((PKG_RELEASE-MAX_REVISIONS))
+    PKG_TO_DELETE=${PKG/$PKG_VERSION-$PKG_REVISION/$PKG_VERSION-$REVISION_TO_DELETE}
+    debug "Pruning obsolete revision ${PKG_VERSION}-${REVISION_TO_DELETE} ..."
```

Comment:
It's already echoed https://github.com/StackStorm/st2-packages/blob/566a81999c83738d320aaad2125b83186310b891/.circle/packagecloud.sh#L136

Bintray example from CircleCI logs:
![clzukmh 1](https://cloud.githubusercontent.com/assets/1533818/13057228/6d039638-d423-11e5-9488-9e6cfa821b7e.png)
https://circleci.com/gh/StackStorm/st2-packages/413